packages/@aws-cdk/aws-apigateway/lib/methodresponse.ts

```diff
@@ -0,0 +1,21 @@
+export interface MethodResponse {
+
+    /**
+     * The method response's status code, which you map to an IntegrationResponse.
+     */
+    statusCode: string;
+
+    /**
+     * Response parameters that API Gateway sends to the client that called a method.
+     * Specify response parameters as key-value pairs (string-to-Boolean maps), with
+     * a destination as the key and a Boolean as the value. Specify the destination
+     * using the following pattern: method.response.header.name, where the name is a
+     * valid, unique header name. The Boolean specifies whether a parameter is required.
+     */
+    responseParameters?: { [destination: string]: boolean };
+
+    /**
+     * The method response's status code, which you map to an IntegrationResponse.
```

Comment:
This is a mapping of content type to model name, not the status code.

Reply:
Totally grabbed the wrong description from the docs 😛 Will fix that.